multi_label_trainer_tfidf.ipynb

```diff
@@ -63,9 +63,10 @@
         "from sklearn.model_selection import train_test_split\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        predicted_proba = shallow_mlp_model(text_batch)[0]
</pre><p> </p><p>If we directly call <code>.predict()</code>  we don't need to separately call <code>numpy()</code>  and won't have to worry about if our model is running in inference mode. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/soumik12345/multi-label-text-classification/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>Now, we can add suggestions and other observations. Something like:</p><p> </p><p>The prediction results are not that great but not below the par for a simple model like ours. We can improve this performance with models that consider word order like LSTM or even those that use Transformers. </p><p> </p><p>It's just a hint, feel free to modify in your own words. </p>

Reply:
@soumik12345 only the above comment and then we can merge.

Reply:
@sayakpaul updated